extensions/amp-story-auto-analytics/0.1/auto-analytics-configs.js

```diff
@@ -68,4 +63,18 @@ export const buildGtagConfig = (gtagId) => ({
       },
     },
   },
+  'linkers': {
+    'ampStoryAutoAnalyticsLinker': {
+      'ids': {
+        'cid': '${clientId}',
+      },
+      'enabled': true,
+      'proxyOnly': false,
+    },
+  },
+  'cookies': {
+    'ampStoryAutoAnalyticsCookies': {
+      'value': 'LINKER_PARAM(ampStoryAutoAnalyticsLinker, cid)',
+    },
+  },
```

Comment:
Not sure if you've done it already, but it would probably be a good idea to add some comments describing what this config does, just so that readers don't have a to have a thorough understanding of amp-analytics.